source/style-guide/style/punctuation.txt

```diff
@@ -60,6 +60,14 @@ Use the following guidelines for colons:
 - If a title or heading includes a colon, capitalize the first word
   that follows the colon, regardless of its part of speech.
 
+- "Lowercase or capital letter after a colon. When a colon is used within
+  a sentence, as in the first two examples in 6.61, the first word following
```

Comment:
Suggestion:

Rather than referencing examples that the reader cannot see, could we reframe this bullet so it's not a direct quote, but instead keep the same ideas and include our own examples?